File: C++/dicecup.cpp
#include <iostream>
#include <algorithm>
#include <cstdlib>
using namespace std;

int main() {
  int n, m;
  cin >> n >> m;

  if (n == m) {
    cout << n + 1 << endl;
  } else {
    int start = min(n, m) + 1;
    int end = abs(n - m) + 1;

    for (int i = start; i < start + end; i++) {
      cout << i << endl;
    }
  }
}

File: Java/dicecup.java
import java.io.*;
import java.util.*;

public class dicecup {
  
  public static void main(String[] args) throws IOException {
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    StringTokenizer st = new StringTokenizer(br.readLine());

    int n = Integer.parseInt(st.nextToken()); 
    int m = Integer.parseInt(st.nextToken());

    if (n == m) System.out.println(n + 1);
    else {
      int start = Math.min(n, m) + 1;
      int end = Math.abs(n - m) + 1;

      for (int i = start; i < start + end; i++) {
        System.out.println(i);
      }
    }
  }
}
